tests/components/palazzetti/conftest.py

```diff
@@ -0,0 +1,56 @@
+"""Fixtures for Palazzetti integration tests."""
+
+from unittest.mock import AsyncMock, patch
+
+import pytest
+
+from homeassistant.components.palazzetti.const import DOMAIN
+from homeassistant.const import CONF_HOST, CONF_MAC, CONF_NAME
+
+from tests.common import MockConfigEntry
+
+
+@pytest.fixture
+def mock_config_entry() -> MockConfigEntry:
+    """Return the default mocked config entry."""
+    return MockConfigEntry(
+        title="palazzetti",
+        domain=DOMAIN,
+        data={
+            CONF_NAME: "Stove",
+            CONF_HOST: "127.0.0.1",
+            CONF_MAC: "11:22:33:44:55:66",
+        },
+        unique_id="unique_id",
+    )
+
+
+@pytest.fixture
+def mock_palazzetti():
+    """Return a mocked PalazzettiClient."""
+    with (
+        patch(
+            "homeassistant.components.palazzetti.coordinator.PalazzettiClient",
+            AsyncMock,
+        ) as client,
+    ):
```

Comment:
Recommend checking out how we patch the Airgradient client in airgradient. We patch both the normal client as the config flow ones. This gets rid of the `patch` in the `test_config_flow.py`